pandas/tests/indexes/multi/test_duplicates.py

```diff
@@ -337,3 +338,16 @@ def test_multi_drop_duplicates_pos_args_deprecation():
         result = idx.drop_duplicates("last")
     expected = MultiIndex.from_arrays([[2, 3, 1], [2, 3, 1]])
     tm.assert_index_equal(expected, result)
+
+
+def test_midx_unique_ea_dtype():
+    # GH#48335
+    a = Series([1, 2, NA, NA], dtype="Int64")
+    b = np.array([1, 2, 3, 3])
+    midx = MultiIndex.from_arrays([a, b], names=["a", "b"])
+    result = midx.unique()
+
+    a = Series([1, 2, NA], dtype="Int64")
```

Comment:
nitpick, can you avoid 1-letter variable names

Reply:
Yep, thx. Stupid copy paste miss